dags/openshift_nightlies/releases/4.7/rosa/default/install.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Much of these defaults have no use in rosa. I.e. we dont change the master instance type, etc. Can we remove the unneeded ones or will that break things?

Reply:
same than https://github.com/cloud-bulldozer/airflow-kubernetes/pull/39#discussion_r693764207

I cannot confirm that deleting those vars will affect or not the benchmarking

Reply:
all removed